packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.Customization.Example.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from 'react';
+import { Button, ButtonType } from 'office-ui-fabric-react/lib/Button';
+import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';
+import { css, autobind, } from 'office-ui-fabric-react/lib/Utilities';
+import styles = require('./CommandBar.Example.scss');
+import { ContextualMenu, IContextualMenuProps, IContextualMenuItem, hasSubmenuItems, DirectionalHint } from '../../ContextualMenu';
+
+export interface SplitDropDownButtonState {
```

Comment:
This is failing checkin, since interfaces should prefix with I

Warning - tslint - src/components/CommandBar/examples/CommandBar.Customization.Example.tsx(8,18): error interface-name: interface name must start with a capitalized I